platforms/android/res/values/localytics.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Define your Localytics app key-->
+    <string name="ll_app_key" translatable="false">MY_APP_KEY</string>
+
+    <!-- Toggle the Localytics referral receiver which is used to attribute sessions to
+    their installation source -->
+    <bool name="ll_referral_receiver_enabled">true</bool>
+
+    <!-- Toggle the Localytics push tracking activity which is used to track the push opened event
+    as well as ensure deeplinking behavior -->
+    <bool name="ll_push_tracking_activity_enabled">true</bool>
+
+    <!-- Specify the GCM Sender ID which will be used to register for push tokens -->
+    <string name="ll_gcm_sender_id" translatable="false">MY_GCM_SENDER_ID</string>
+
+    <!-- Toggle the Localytics FirebaseInstanceIdService and FirebaseMessagingService, which handle
+    incoming requests from FCM. If this value is set to true, then the app must specify the
+    com.google.firebase:firebase-messaging dependency-->
+    <bool name="ll_fcm_push_services_enabled">false</bool>
+
+    <!-- Toggle the Localytics LocationUpdateReceiver and BootReceiver for Places integrations
+    If this value is set to true then the app must specify the
+    com.google.android.gms:play-services-location dependency-->
+    <bool name="ll_places_enabled">true</bool>
+
+    <!-- Configure the Localytics session timeout -->
+    <integer name="ll_session_timeout_seconds">15</integer>
+
+    <!--Configure the default channel used for push notifications-->
+    <string name="ll_default_push_channel_id" translatable="false">localytics_default</string>
+    <!--Configure the default channel name used for push notifications-->
+    <string name="ll_default_push_channel_name" translatable="false">Default</string>
+    <!--Configure the default channel description used for push notifications-->
+    <string name="ll_default_push_channel_description" translatable="false"/>
+
+    <!--Configure the default channel used for places notifications-->
+    <string name="ll_default_places_channel_id" translatable="false">localytics_default</string>
+    <!--Configure the default channel name used for places notifications-->
+    <string name="ll_default_places_channel_name" translatable="false">Default</string>
+    <!--Configure the default channel description used for places notifications-->
+    <string name="ll_default_places_channel_description" translatable="false"/>
```

Comment:
This is missing the location update settings (prob need to grab the latest localytics.xml)

Reply:
They aren't required, so I wouldn't worry about it.